### PR TITLE
Make everything compile in JS & Native

### PR DESCRIPTION
**Pull request by Lyxal**
+ make the print function not println
+ make the native workflow actually run
  (for some reason, it didn't want to recognise the `env`)

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -18,6 +18,8 @@ rewrite.imports.sort = original
 rewrite.imports.groups = [["vyxal\\..*"]]
 rewrite.imports.contiguousGroups = no
 
+rewrite.trailingCommas.style = keep # frick you removing my commas.
```
Comment (ysthakur) [inline .scalafmt.conf:21]: We could even make this always

File: native/src/main/scala/Main.scala [hunk 2]
```diff
@@ -4,4 +4,8 @@ import vyxal.cli.CLI
 
 object Main:
   def main(args: Array[String]): Unit =
-    CLI.run(args)
+    if args.nonEmpty && args(0).charAt(0) != '-' then
```
Comment (ysthakur) [inline native/src/main/scala/Main.scala:7]: This logic should be handled in `CLI`

File: shared/src/main/scala/cli/CLI.scala [hunk 3]
```diff
@@ -31,7 +28,7 @@ object CLI:
       settings: Settings = Settings()
   )
 
-  def run(args: Array[String]): Unit =
+  def run(args: Array[String], program: Option[String]): Unit =
```
Comment (ysthakur) [inline shared/src/main/scala/cli/CLI.scala:31]: Is scopt not parsing code given directly correctly?

**Comment by ysthakur** [comment]
Sorry, I didn't even realize that the workflows weren't running

**Comment by ysthakur** [comment]
If I understand correctly, the only reason to split up the code was because `java.io.File` didn't compile in JS, so I modified it to take the file name as a string instead of taking a `File`. Also modified `execute` to take a boolean `literate` parameter to avoid all the `if`s everywhere.

File: jvm/src/main/scala/Main.scala [hunk 4]
```diff
@@ -2,6 +2,8 @@ package vyxal
 
 import vyxal.cli.CLI
 
+import java.io.File
```
Comment (ysthakur) [inline jvm/src/main/scala/Main.scala:5]:
```suggestion
```

File: jvm/src/main/scala/Main.scala [hunk 5]
```diff
@@ -2,6 +2,7 @@ package vyxal
 
 import vyxal.cli.CLI
 
+
```
Comment (ysthakur) [inline jvm/src/main/scala/Main.scala:5]:
```suggestion
```

**Comment by ysthakur** [comment]
Do vyxal.js and vyxal.js.map have to be added to git? I feel like we could upload them as artifacts or something

**Review by ysthakur — APPROVED** [review]
My bad, didn't see that the JS didn't compile. I don't know why the native workflow doesn't evaluate the matrix.os variable, but maybe that happens only if it's on version-3?

**Review by Steffan153 — APPROVED** [review]

**Pull request merged**